index.js

```diff
@@ -98,106 +103,114 @@ module.exports = {
+    "no-sequences": ["warn"],
+    "react/prop-types": [
       1,
       {
         skipUndeclared: true,
       },
     ],
-    'react/self-closing-comp': 'error',
-    'strict': 0,
-    'react-hooks/rules-of-hooks': 'error',
-    'react-hooks/exhaustive-deps': 'warn',
-    'no-unused-vars': 'error',
-    'no-multiple-empty-lines': [
-      'error',
+    "react/self-closing-comp": "error",
+    strict: 0,
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+    "no-unused-vars": "error",
+    "no-multiple-empty-lines": [
+      "error",
       {
-        'max': 1,
-        'maxEOF': 1
-      }
+        max: 1,
+        maxEOF: 1,
+      },
     ],
-    'space-unary-ops': 'error',
-    'jest/no-disabled-tests': 'warn',
-    'jest/no-focused-tests': 'error',
-    'jest/prefer-to-have-length': 'warn',
-    'react/react-in-jsx-scope': 'off',
-    'jest/valid-expect': 'off',
-    'no-trailing-spaces': 'off',
-    'jest/no-identical-title': 'off',
-    'no-shadow': 'off',
-    'eqeqeq': 'off',
-    'consistent-return': 'off',
-    'no-unused-expressions': 'off',
-    'import/namespace': ['error', { allowComputed: true }],
-    'import/no-unresolved': 'warn',
-    'simple-import-sort/imports': [
-      'error',
+    "space-unary-ops": "error",
+    "jest/no-disabled-tests": "warn",
+    "jest/no-focused-tests": "error",
+    "jest/prefer-to-have-length": "warn",
+    "react/react-in-jsx-scope": "off",
+    "jest/valid-expect": "off",
+    "no-trailing-spaces": "off",
+    "jest/no-identical-title": "off",
+    "no-shadow": "off",
+    eqeqeq: "off",
+    "consistent-return": "off",
+    "no-unused-expressions": "off",
+    "import/namespace": ["error", { allowComputed: true }],
+    "import/no-unresolved": "warn",
+    "simple-import-sort/imports": [
+      "error",
       {
         groups: [
           // Packages `react` related packages come first.
-          ['^react', '^[a-z]'],
+          ["^react", "^[a-z]"],
           // Packages from a '@deriv' scope come second.
-          ['^@\\w'],
+          ["^@\\w"],
           // Absolute imports and other imports from aliases like 'Components/...'
-          ['^[A-Z]'],
+          ["^[A-Z]"],
           // Internal packages.
-          ['^(@|components)(/.*|$)'],
+          ["^(@|components)(/.*|$)"],
           // Parent imports. Put `..` last.
-          ['^\\.\\.(?!/?$)', '^\\.\\./?$'],
+          ["^\\.\\.(?!/?$)", "^\\.\\./?$"],
           // Other relative imports. Put same-folder imports and `.` last.
-          ['^\\./(?=.*/)(?!/?$)', '^\\.(?!/?$)', '^\\./?$'],
+          ["^\\./(?=.*/)(?!/?$)", "^\\.(?!/?$)", "^\\./?$"],
           // Style imports.
-          ['^.+\\.?(css)$'],
+          ["^.+\\.?(css)$"],
         ],
       },
     ],
-  
   },
-  
+  "linebreak-style": ["error", "unix"],
+  "jsx-quotes": ["error", "prefer-single"],
+  "no-confusing-arrow": ["error", { allowParens: false }],
```

Comment:
do we need this? 🤔 it will throws lots of error with the current code we have in both and deriv-app I guess.

Reply:
just ignore this comment

Reply:
Thanks @shayan-deriv